python_src/src/lamp_py/tableau/jobs/rt_rail.py

```diff
@@ -26,7 +26,6 @@ def __init__(self) -> None:
             "   date(vt.service_date::text) as service_date"
             "   , TIMEZONE('UTC', TO_TIMESTAMP(extract(epoch FROM date(vt.service_date::text)) + vt.start_time)) as start_datetime"
             "   , TIMEZONE('UTC', TO_TIMESTAMP(extract(epoch FROM date(vt.service_date::text)) +  vt.static_start_time)) as static_start_datetime"
-            "   , ve.pm_trip_id"
             "   , ve.stop_sequence"
```

Comment:
I don't think `pm_trip_id` or `updated_on` fields have any use to OPMI, so drop from parquet file.